src/relation_handlers.py

```diff
@@ -33,9 +33,9 @@
 
 import microceph
 from ceph import get_osd_count
-from ceph_broker import Capabilities
+from ceph_broker import Capabilities, get_named_key
```

Comment:
we should not import functionality from ceph broker given that we eventually want to deprecate it. If it makes sense (is not anti ops) we should refactor out such functionality into relevant files (`[ceph.py, microceph.py, <relation>.py]`.

Reply:
Leaving this import as is for now, but I've moved other added functions to their "right" place. Will leave the ceph broker refactor to its own separate PR.